NEWS.md

```diff
@@ -1,3 +1,8 @@
+# etn 2.2.1
+
+- Fix bug in `write_dwc()` bug where the function would return an error due to an updated dependency. #293
```

Comment:
```suggestion
* Fix bug in `write_dwc()` where the function would return an error due to an updated dependency (#293).
```

Typo ("bug" twice) + issue number is placed between parenthesis before end period. Since `*` is used for list bullets before, I would keep doing that here.